packages/arb-token-bridge-ui/src/util/withdrawals/fetchETHWithdrawalsFromEventLogs.ts

```diff
@@ -11,20 +11,24 @@ import { L2ToL1MessageReader } from '@arbitrum/sdk'
  * @param query.l2Provider Provider for the L2 network
  */
 export function fetchETHWithdrawalsFromEventLogs({
-  toAddress,
+  sender,
```

Comment:
Hmm doesn't `fromAddress` map to `sender`, and `toAddress` map to `receiver`, ideally?

Reply:
Yeah you are right. Although it won't impact the functionality because sender and receiver should always be the same address, it makes more sense to have receiver here.

We can't have sender yet because the sdk doesn't support it.

Reply:
Yeah you are right. This should be destinationAddress which is the receiver. Sender is not supported by eth event logs yet.